tensorflow_datasets/core/lazy_imports_lib.py

```diff
@@ -52,106 +50,106 @@ class LazyImporter(object):
 
   @utils.classproperty
   @classmethod
-  def apache_beam(cls: Type[T]) -> object:
+  def apache_beam(cls) -> ModuleType("apache_beam"):
     return _try_import("apache_beam")
 
   @utils.classproperty
   @classmethod
-  def crepe(cls: Type[T]) -> object:
+  def crepe(cls) -> ModuleType("crepe"):
     return _try_import("crepe")
 
   @utils.classproperty
   @classmethod
-  def cv2(cls: Type[T]) -> object:
+  def cv2(cls) -> ModuleType("cv2"):
```

Comment:
This should be `ModuleType`, not `ModuleType(xyz)`. `ModuleType` is the `type(module)` while `ModuleType(xyz)` is an instance of module.

Typing annotation indicates the type.

Reply:
Thanks for clarification. I understand the concept now. I will change that.